src/table.js

```diff
@@ -672,7 +678,8 @@ export const createTable = (
   schema,
   rowsCount = 3,
   colsCount = 3,
-  withHeaderRow = true
+  withHeaderRow = true,
+  withDefaultNonWidthChar = false
```

Comment:
here we make assumptions about people's schema:
* "paragraph" might be called differently, e.g. "p"
* people might put any other node inside table cell in schema, something like "cellWrapper" node.

I suggest passing a ProseMirror node as a content of a cell instead of "withDefaultNonWidthChar" prop, so that you can create a cell with the provided custom content of the type you need.

Reply:
Great point, I'll work on this